test/test_save.py

```diff
@@ -0,0 +1,36 @@
+import os
+
+import py.path
+import rasterio
+
+from satsense import FeatureVector, Image
+from satsense.extract import extract_feature
+from satsense.features import HistogramOfGradients
+from satsense.generators import FullGenerator
+
+
+def test_netcdf_save(image, tmpdir):
+    netcdf_save(image, tmpdir)
+
+
+def netcdf_save(image, tmpdir, extension='nc'):
+    feature = HistogramOfGradients([(25, 25), (50, 50)])
+    generator = FullGenerator(image, (25, 25))
```

Comment:
It would be better to test this without using a generator and the extract_feature function, to decouple the implementation a bit. If you simply define the values of the feature vector in this test, that would have the added advantage that you can check that they are stored in the correct way, i.e. not mixed up, see also https://github.com/DynaSlum/satsense/pull/39/files#r240687268

Reply:
I removed the test_save.py file from the repo as what it tests is covered by the round-trip load.